### PR TITLE
add a popup when user copy in address and received page and also adde…

### PR DESCRIPTION
**Pull request by vikas-cis**
…d paste functionality on send page

### CODE REVIEW

File: src/app/wallet/send/send.component.html [hunk 1]
```diff
@@ -73,7 +73,7 @@ <h5 fxFlex="16.6">To Account</h5>
               <md-radio-button value="anon_balance" disabled="{{ send.input === 'anon_balance' }}" fxFlex="16.6">Private</md-radio-button>
               <div fxFlex="50">
                 <span class="balance-label">Balance: </span>
-                <span class="balance-amount enable-select">{{ getBalance(send.input) }}</span>
+                <span class="balance-amount enable-select">{{ getBalance(send.output) }}</span>
```
Comment (kewde) [inline src/app/wallet/send/send.component.html]: why change from input -> output type?

Comment (vikas-cis) [reply src/app/wallet/send/send.component.html]: the balance showing same before

Comment (kewde) [reply src/app/wallet/send/send.component.html]: The balance will indeed show the same as before, but it's non-nonsensical as the input type defines the balance. Sure in the balance transfer tab they're both the same but still, doesn't make sense.

Comment (vikas-cis) [reply src/app/wallet/send/send.component.html]: if you see the modal name are different inside two cards. That the reason we need to pass the output rather then input

Comment (kewde) [reply src/app/wallet/send/send.component.html]: This used to work, indeed it's supposed to be output.

Comment (vikas-cis) [reply src/app/wallet/send/send.component.html]: thanks

**Comment by kewde** [comment]
merge conflicts but other than that the code looks good

**Pull request merged**